commit f73337355e47274141a29b013f024c1c1f47ddf0
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Thu Jan 24 15:09:28 2019 +0000

    clk: clk-bcm2835: Use %zd when printing size_t
    
    The debug text for how many clocks have been registered
    uses "%d" with a size_t. Correct it to "%zd".
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/clk/bcm/clk-bcm2835.c b/drivers/clk/bcm/clk-bcm2835.c
index a142253afdff..3e001a7467a0 100644
--- a/drivers/clk/bcm/clk-bcm2835.c
+++ b/drivers/clk/bcm/clk-bcm2835.c
@@ -2271,7 +2271,7 @@ static int bcm2835_clk_probe(struct platform_device *pdev)
 		return ret;
 
 	/* note that we have registered all the clocks */
-	dev_dbg(dev, "registered %d clocks\n", asize);
+	dev_dbg(dev, "registered %zd clocks\n", asize);
 
 	return 0;
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1323_f73337355e47274141a29b013f024c1c1f47ddf0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1323_f73337355e47274141a29b013f024c1c1f47ddf0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
